Sources/Extensions/SwiftStdlib/DictionaryExtensions.swift

```diff
@@ -30,8 +30,8 @@ public extension Dictionary {
     ///		dict.keys.contains("key2") -> false
     ///
     /// - Parameter keys: keys to be removed
-    public mutating func removeAll(keys: [Key]) {
-        keys.forEach({ removeValue(forKey: $0)})
+    public mutating func removeAll<S: Sequence>(keys: S) where S.Element == Key {
+        Set<Key>(keys).forEach({ removeValue(forKey: $0) })
```

Comment:
I think SwiftLint gives a warning when you can remove the parenthesis, e.g. in `forEach`. So it would just be
`forEach { removeValue(forKey: $0) }`